platforms/chibios/_pin_defs.h

```diff
@@ -292,3 +292,63 @@
 #    define K14 PAL_LINE(GPIOK, 14)
 #    define K15 PAL_LINE(GPIOK, 15)
 #endif
+
+#if defined(MCU_LPC)
+//   GPIO Port 0
+#    define P0_0 PAL_LINE(IOPORT0, 0)
```

Comment:
This should probably be handled like https://github.com/qmk/qmk_firmware/blob/develop/platforms/chibios/vendors/RP/_pin_defs.h instead of creating a file full of platform ifdefs.